vnext/Microsoft.ReactNative/Fabric/Composition/Modal/WindowsModalHostViewComponentView.cpp

```diff
@@ -38,43 +48,44 @@ winrt::Microsoft::ReactNative::ComponentView WindowsModalHostComponentView::Crea
   return winrt::make<WindowsModalHostComponentView>(compContext, tag, reactContext);
 }
 
-// constants for creating a new windows (code mostly taken from LogBox)
+// constants for creating a new windows
 constexpr PCWSTR c_modalWindowClassName = L"MS_REACTNATIVE_MODAL";
 constexpr auto CompHostProperty = L"CompHost";
-const int MODAL_DEFAULT_WIDTH = 500;
-const int MODAL_DEFAULT_HEIGHT = 500;
+const int MODAL_MIN_WIDTH = 50;
+const int MODAL_MIN_HEIGHT = 50;
+
+float ScaleFactor(HWND hwnd) noexcept {
```

Comment:
nit: We seem to call for ScaleFactor throughout the codebase, maybe it's time to make this a shared utility function?

Reply:
hmmm looked through the code-base and it only gets used when creating a new hwnd which is in 4 spots in our code-base (3 of which are just creating a new app). I don't expect for another component to use this but if it does we can add it to a shared file!